src/utils/token-operations.util.ts

```diff
@@ -4,6 +4,8 @@ import { ActivityTypeEnum } from '../enums/activity-type.enum';
 import { OperationFa12Interface, OperationFa2Interface, OperationInterface } from '../interfaces/operation.interface';
 import { isDefined } from './is-defined';
 
+export const getOperationGroupByHash = <T>(hash: string) => tzktApi.get<Array<T>>(`operations/${hash}`);
```

Comment:
it's extremely hard to read all those queries below.  
rewritethem using axios `config.params`